cst438_gradebook/src/main/java/com/cst438/controllers/EnrollmentController.java

```diff
@@ -32,8 +32,21 @@ public class EnrollmentController {
 	public EnrollmentDTO addEnrollment(@RequestBody EnrollmentDTO enrollmentDTO) {
 		
 		//TODO  complete this method in homework 4
-		
-		return null;
+		// this creates a new Enrollment to store the student's enrollment in the enrollmentRepository
+		Enrollment e = new Enrollment();
+		e.setStudentEmail(enrollmentDTO.studentEmail);
+		e.setStudentName(enrollmentDTO.studentName);
+		Course c = courseRepository.findById(enrollmentDTO.course_id).orElse(null);
+		// this if statement checks if the course stated in the variable enrollmentDTO is in the database
+		// if it is not then an error is thrown and a message is displayed in the console on why the error has occurred
```

Comment:
The comment placement is a little strange. comment on line 40 should go before line 39.